runtimes/common/src/constants.rs

```diff
@@ -218,27 +228,50 @@ pub mod did {
 	pub const MAX_NUMBER_OF_URLS_PER_SERVICE: u32 = 1;
 }
 
-pub mod treasury {
-	use crate::{Balance, BlockNumber};
+pub mod did_lookup {
+	use super::*;
 
-	use super::{BLOCKS_PER_YEAR, KILT};
+	pub const CONNECTION_SIZE: u32 = 80;
+	pub const DID_CONNECTION_DEPOSIT: Balance = deposit(1, CONNECTION_SIZE);
+}
+
+pub mod treasury {
+	use super::*;
 
 	pub const INITIAL_PERIOD_LENGTH: BlockNumber = BLOCKS_PER_YEAR.saturating_mul(5);
 	const YEARLY_REWARD: Balance = 2_000_000u128 * KILT;
 	pub const INITIAL_PERIOD_REWARD_PER_BLOCK: Balance = YEARLY_REWARD / (BLOCKS_PER_YEAR as Balance);
 }
 
 pub mod proxy {
-	pub const MAX_PROXIES: u16 = 10;
-	pub const MAX_PENDING: u16 = 10;
+	use super::*;
+
+	parameter_types! {
+		// One storage item; key size 32, value size 8; .
```

Comment:
Do we need to specify the key size?

Reply:
Is it bothering you? The comment is taken from parity. Together with the values for the deposit function. I added them because they are nice to have and come with no cost.

Reply:
I understand, it just seems inconsistent to me since we don't mention key sizes anywhere else, and I am not sure whether it affects anything or not.